src/ui/osx/TogglDesktop/IdleEvent.h

```diff
@@ -14,4 +14,9 @@
 @property (nonatomic, copy) NSString *since;
 @property (nonatomic, copy) NSString *duration;
 @property (nonatomic, copy) NSString *timeEntryDescription;
+@property (nonatomic, copy) NSString *projectName;
+@property (nonatomic, copy) NSString *taskName;
+@property (nonatomic, copy) NSColor *projectColor;
```

Comment:
I suggest that we should use `strong` on NSColor rather than `copy`. Ideally, `copy` is only suitable for `NSString`. Otherwise, it's `strong` attribute